tests/usdt_sample/scripts/latency.c

```diff
@@ -0,0 +1,127 @@
+#include <linux/blkdev.h>
+#include <uapi/linux/ptrace.h>
+
+// The following methods were taken from the verbose output of argdist.py.
+// Function names were adjusted to match the BPF function name expectations.
+static inline int _bpf_readarg_trace_operation_start_1(struct pt_regs* ctx, void* dest, size_t len)
```

Comment:
@bveldhoen: you don't need to pass debug=4 to get the generated code. @yonghong-song suggested using this flag to _see_ the generated code so it's easier to understand what's going on. If you don't get the generated code by default, that's a bug.